### ISSUE TITLE
Xcode 12 compile error for simulator

### ISSUE THREAD

**Issue by voltec** [NextLevel/NextLevel#233]
Needs to add `#if !( targetEnvironment(simulator) )` here:
<img width="1086" alt="Screen Shot 2020-09-22 at 23 11 39" src="https://user-images.githubusercontent.com/3304776/93926930-34d3be80-fd29-11ea-8d79-a3e0ad05817e.png">

<img width="617" alt="Screen Shot 2020-09-22 at 23 11 50" src="https://user-images.githubusercontent.com/3304776/93926950-3ef5bd00-fd29-11ea-8406-c466402525bb.png">

**Comment by piemonte** [comment]
@voltec this was fixed in the latest release

📎 https://github.com/NextLevel/NextLevel/issues/229

**Comment by jeremyschoenherr** [comment]
hi! i'm assuming this is fixed in 0.17.0, but when i try and pull that version via cocoapods its saying that it couldn't find it:

```
[!] CocoaPods could not find compatible versions for pod "NextLevel":
  In Podfile:
    NextLevel (~> 0.17.0)
```

this after running `pod repo update`

any suggestions or is the release still propagating out?
